### DISCUSSION TITLE
Custom Js and Custom Css file not loading on cart and checkout page

### DISCUSSION THREAD

**Discussion by JamesDenish** [breezefront/community#194] [1. Help]
I have made custom js and css file in breeze theme which is loading everywhere in my website but not on cart and checkout page and console does not show any error regarding those custom files.

@vovayatsyuk can you please help me out from this as of now I am trying but nothing is working proper on cart and checkout page.

Please help and thanks in advance......!!!!!

**Comment by JamesDenish** [comment]
@vovayatsyuk  Do you have any idea ?

**Comment by vovayatsyuk** [comment]
1. Custom js file is not included on non-breeze pages (checkout and other manually excluded pages). You have to duplicate the same logic in another js file that will be included in the luma-based theme. See how we did that for breeze-blank: https://github.com/breezefront/theme-frontend-breeze-blank/blob/master/requirejs-config.js
2. Custom CSS - I don't know why it's happening. How do you add your CSS? See https://breezefront.com/docs/custom-styles.

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

Please reply it's urgent I want to add slider on cart and checkout page in menu and  I am getting the below error in console  in my custom js file 

Uncaught TypeError: $(...).parent(...).css(...).pagebuilderSlider is not a function

Please help it's little bit urgent.

Thanks

**Reply by vovayatsyuk** [reply]
When Breeze is disabled you have a breeze theme with require-js scripts from Magento, so you have to implement everything using Magento scripts, just like you would do in Luma.

**Comment by JamesDenish** [comment]
Okay. Thanks for replying.

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

I have followed the step you told to implement but it is giving error for navpro.js file.

Can you please help or reply for solution

Thanks

**Reply by vovayatsyuk** [reply]
You need to create a `pagebuilderSlider` widget for requirejs-based pages so it will not break when the breeze is disabled.

Here is a first manual from the web: https://kiwicommerce.co.uk/how-to-create-a-jquery-widget-in-magento-2/

**Comment by JamesDenish** [comment]
Hello @vovayatsyuk 

We have implemented the above steps we got the slider view but we are not getting arrows or sliding in movable position so can you suggest for the same.

Please help me out. 
Thanks

**Comment by JamesDenish** [comment]
![image](https://github.com/breezefront/community/assets/142228975/b86f37a4-b42a-4c42-9131-1f02edd2cb19)
 Above is the screenshot the previous question I asked for.
 
 Thanks
